packages/backend-modules/access/graphql/schema.js

```diff
@@ -44,6 +44,21 @@ type mutations {
     campaignId: ID!
     payload: JSON
   ): AccessGrant!
+
+  """
+  Terminate access before access grant ends
+  """
+  terminateAccess(id: ID!): Boolean!
+
+  """
+  Change access grant
+  """
+  changeAccess(
+    id: ID! 
+    revoke: Boolean!
+    invalidate: Boolean! 
+    noFollowup: Boolean!
+  ): Boolean!
 }
```

Comment:
This mutation seems to be an All-Inclusive buffet. (Boolean flags allow revoking and invalidating at once?)

Urging to split into `revokeAccess(...)` (exists already) and `invalidateAccess(noFollowUp)`.